package.json

```
@@ -10,19 +10,22 @@
     "npm": ">=8.19.3 <9"
   },
   "devDependencies": {
-    "@wordpress/env": "^5.7.0",
-    "@wordpress/scripts": "^24.6.0"
+    "@wordpress/env": "^8.11.0",
+    "@wordpress/scripts": "^24.6.0",
+    "patch-package": "^8.0.0"
   },
   "scripts": {
+    "postinstall": "patch-package",
     "wp-env": "wp-env",
     "lint-js": "wp-scripts lint-js",
     "format-js": "npm run lint-js -- --fix",
-    "prelint-php": "wp-env run composer 'install --no-interaction'",
-    "phpstan": "wp-env run composer run-script phpstan",
-    "lint-php": "wp-env run composer run-script lint",
-    "format-php": "wp-env run composer run-script format",
-    "pretest-php": "wp-env run composer 'install --no-interaction'",
-    "test-php": "wp-env run phpunit 'phpunit -c /var/www/html/wp-content/plugins/$(basename $(pwd))/phpunit.xml.dist --verbose'",
-    "test-php-multisite": "wp-env run phpunit 'WP_MULTISITE=1 phpunit -c /var/www/html/wp-content/plugins/$(basename $(pwd))/tests/phpunit/multisite.xml --verbose'"
+    "prelint-php": "wp-env run tests-cli --env-cwd=wp-content/plugins/plugin-check/ composer install --no-interaction",
```

```suggestion
    "prelint-php": "wp-env run tests-cli --env-cwd=wp-content/plugins/$(basename $(pwd))/ composer install --no-interaction",
```

This change will again introduce #181 so we should update it. The change applies to all the commands.

---

This could just be changed to `composer install`, no?

---

Actually, we can completely remove this line

---

Why can it be removed?

---

Because PHPCS is configured in the build-cs directory, no dependencies need to be installed in the root.

Running `composer phpcs` automatically installs it in build-cs.